### PR TITLE
Add a fence.i vplan for 40x

### PR DESCRIPTION
**Pull request by silabs-robin**
Attempting to fulfill this https://github.com/openhwgroup/core-v-verif/issues/871 task.
This PR adds a vplan for `fence.i`.

### CODE REVIEW

**Review by MikeOpenHWGroup — APPROVED** [review]

**Comment by silabs-robin** [comment]
Hi @MikeOpenHWGroup , sorry my bad, I forgot to apply a "do not merge". My intention was to wait until after the review.
Btw, any idea why it appears I don't have permissions to add labels/assignees/reviewers/etc? I get that I don't have merge permissions, but these other things seems harmless.

**Pull request merged**
